modules/deploy/partials/kubernetes/guides/start-streaming.adoc

```diff
@@ -40,22 +124,21 @@ SUMMARY
 NAME        twitch_chat
 PARTITIONS  1
 REPLICAS    1
+
 CONFIGS
 =======
-KEY                           VALUE       SOURCE
-cleanup.policy                delete      DYNAMIC_TOPIC_CONFIG
-compression.type              producer    DEFAULT_CONFIG
-max.message.bytes             1048576     DEFAULT_CONFIG
-message.timestamp.type        CreateTime  DEFAULT_CONFIG
-redpanda.remote.delete        true        DEFAULT_CONFIG
-redpanda.remote.read          false       DEFAULT_CONFIG
-redpanda.remote.write         false       DEFAULT_CONFIG
-retention.bytes               -1          DEFAULT_CONFIG
-retention.local.target.bytes  -1          DEFAULT_CONFIG
-retention.local.target.ms     86400000    DEFAULT_CONFIG
-retention.ms                  604800000   DEFAULT_CONFIG
-segment.bytes                 134217728   DEFAULT_CONFIG
-segment.ms                    1209600000  DEFAULT_CONFIG
+KEY                     VALUE                          SOURCE
+cleanup.policy          delete                         DYNAMIC_TOPIC_CONFIG
+compression.type        producer                       DEFAULT_CONFIG
+message.timestamp.type  CreateTime                     DEFAULT_CONFIG
+partition_count         1                              DYNAMIC_TOPIC_CONFIG
+redpanda.datapolicy     function_name:  script_name:   DEFAULT_CONFIG
+redpanda.remote.read    false                          DEFAULT_CONFIG
+redpanda.remote.write   false                          DEFAULT_CONFIG
+replication_factor      1                              DYNAMIC_TOPIC_CONFIG
+retention.bytes         -1                             DEFAULT_CONFIG
+retention.ms            604800000                      DEFAULT_CONFIG
+segment.bytes           1073741824                     DEFAULT_CONFIG
```

Comment:
Interesting why config has changed.

Reply:
In my first test, I used a different replication factor. But decided to keep it simple and use the existing defaults.